src/objects.js

```diff
@@ -1,41 +1,66 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {
+    name: name, 
+    age: age
+  }
 };
 
 const getName = object => {
-  // your code here
+  return object.name
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property]
+  // use square bracket notation to access a property using its variable name 
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  if (object[property]){
+    return true 
+  }
+  return false
 };
 
 const isOver65 = person => {
-  // your code here
+  if (person.age > 65){ // use dot notation to retireve each key value 
+    return true
+  }
+  return false 
 };
 
 const getAges = people => {
-  // your code here
+  const ages = [];
+  for (i = 0; i < people.length; i++) { 
+    if (people[i].age) { 
+      ages.push(people[i].age);
+    }
+  }
+  return ages;
 };
 
 const findByName = (name, people) => {
-  // your code here
-};
+  return people.find(person => person.name === name)
+}
 
 const findHondas = cars => {
-  // your code here
+ return cars.filter(car => car.manufacturer === 'Honda')
 };
 
 const averageAge = people => {
-  // your code here
+  const totalAge = people.reduce((acc, person) => acc + person.age, 0)
+  const averageAge = totalAge / people.length
```

Comment:
You are naming this variable the same as the function. This is throwing an error. You will have to choose uniqe names for all functions/variables for easy understanding and less confusion.
for example: const averageAgeOfPerson
Also you could return this in one line instead of assign total age and average age separately